pkg/sriov/sriov.go

```diff
@@ -126,7 +126,8 @@ type Manager interface {
 	SetupVF(conf *sriovtypes.NetConf, podifName string, cid string, netns ns.NetNS) (string, error)
 	ReleaseVF(conf *sriovtypes.NetConf, podifName string, cid string, netns ns.NetNS) error
 	ResetVFConfig(conf *sriovtypes.NetConf) error
-	ApplyVFConfig(conf *sriovtypes.NetConf) error
+	ApplyVFConfig(conf *sriovtypes.NetConf, pfLink netlink.Link) error
+	FillOriginalVfInfo(conf *sriovtypes.NetConf) (netlink.Link, error)
```

Comment:
the name of the method and what it returns is a bit confusing.
Id prefer that it will just do one thing and fill the original VF info.